tmva/tmva/inc/TMVA/Types.h

```diff
@@ -171,7 +169,7 @@ namespace TMVA {
    private:
 
       Types();
-#if __cplusplus > 199711L && !defined _MSC_VER
+#if !defined _MSC_VER
```

Comment:
@bellenot I noticed that you added this check in 4393d38af2fbedd9061e5a25a8dcdd4db2589188, so I've left `!defined _MSC_VER` for now even though I don't understand what's causing problems here...

Reply:
I'll check. It might be an incompatibility with old versions of Visual Studio. Thanks for pinging me!